### PR TITLE
Blog post: Creating Sourcegraph's product design princples

### PR DESCRIPTION
**Pull request by rebeccadee**
Closes https://github.com/sourcegraph/about/issues/2593

### CODE REVIEW

File: blogposts/2021/creating-sourcegraph-product-design-principles.md [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+
+Now that our team has these product design principles, we’re using them in our work to make better decisions faster. 
+Over time, our team and developers using Sourcegraph in their work will see these principles reflected as these decisions accumulate.
+
+If your company doesn’t have product design principles, consider creating them together! 
+Sourcegraph is an open company, and so our process, artifacts, and resources are all available to you.
+
+**Resources**
+
+*   [Design principles planning and workshop document](https://docs.google.com/document/d/10Ko8z1ozVaUStuPG2-cM_EJUVJvcgabv8qZYNAJXn24/edit#)
```
Comment (rebeccadee) [inline blogposts/2021/creating-sourcegraph-product-design-principles.md:129]: @quinnkeast Can you update the permissions here so people outside of Sourcegraph can view?

File: blogposts/2021/creating-sourcegraph-product-design-principles.md [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+That’s why we prioritize asynchronous collaboration together with thoughtful moments of real-time collaboration.
+
+We wanted to involve product designers, product managers, and engineers throughout the process to co-create our design principles. 
+To make sure we could create the best collaborative environment, we planned a process that would start asynchronously before bringing us together for a real-time workshop when we needed an immediate feedback loop.
+
+
+### The process
+
+
+#### 1. Aligning on our goals and conceptual understanding of principles
```
Comment (rebeccadee) [inline blogposts/2021/creating-sourcegraph-product-design-principles.md:58]: @quinnkeast I'm not sure why these are rendering larger than "The process" – if you know how to correct it please do

**Review by rebeccadee — COMMENTED** [review]
@quinnkeast This is ready for your last pass before we publish 🎉 

One last question, I don't have my terminal set up to do anything on my new laptop. Do you know how to push the hero image to wherever it needs to live for it to be included in the preview when we share the post?

File: blogposts/2021/creating-sourcegraph-product-design-principles.md [hunk 3]
```diff
@@ -0,0 +1,132 @@
+---
+title: "When gut instinct doesn’t scale: Creating Sourcegraph’s product design principles"
+author: Quinn Keast
+authorUrl:
```
Comment (rebeccadee) [inline blogposts/2021/creating-sourcegraph-product-design-principles.md:4]: @quinnkeast Feel free to add your desired URL here 😄

File: blogposts/2021/creating-sourcegraph-product-design-principles.md (hunk 1 above)
Comment (quinnkeast) [reply blogposts/2021/creating-sourcegraph-product-design-principles.md:129]: Updated!

**Review by quinnkeast — APPROVED** [review]

**Pull request merged**